lib/events/encryptedfile.h

```diff
@@ -0,0 +1,84 @@
+// SPDX-FileCopyrightText: 2021 Carl Schwan <carlschwan@kde.org>
+//
+// SPDX-License-Identifier: LGPl-2.1-or-later
+
+#pragma once
+
+#include <QObject>
```

Comment:
```suggestion
```
If it's for `Q_GADGET`, `quotient_common.h` (included via `converters.h`) already includes a necessary Qt header.